src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from 'classnames';
+import React, { useCallback, useMemo } from 'react';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  filterType: FilterType
+  setFilterType: (status: FilterType) => void
+  completedTodos: number
+  todosLength: number
+  onRemove: () => Promise<void>
+};
+
+export const Footer: React.FC<Props> = React.memo(({
+  filterType,
+  setFilterType,
+  completedTodos,
+  onRemove,
+  todosLength,
+}) => {
+  const uncompletedCount = todosLength - completedTodos;
+
+  const filterHrefByType = useMemo(() => ({
+    [FilterType.All]: '#/',
+    [FilterType.Active]: '#/active',
+    [FilterType.Completed]: '#/completed',
+  }), []);
```

Comment:
As for me this object should be in constants file, 'cause there its not depends on props or any component data to store it here